### PR TITLE
[8.0.0] Coding standards, run PHP_CodeSniffer on Travis

### LINKED ISSUE #1176: coding standards - PSR-2?
(this would apply to all PHP projects in the roots org)

### PR DESCRIPTION
**Pull request by retlehs**
* Add Roots coding standards based on PSR-2
* Add PHPCS to Travis
* Update PHP files to pass coding standards

Closes #1176

### CODE REVIEW

**Comment by austinpray** [comment]
lol

File: ruleset.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<ruleset name="Roots">
```
Comment (Ore4444) [inline ruleset.xml]: Sage

Comment (retlehs) [inline ruleset.xml]: the ruleset is for all roots projects, not just sage — this will be added to our other repos

File: ruleset.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  <description>Roots Coding Standards</description>
+
+  <!-- Use PSR-2 as a base -->
+  <rule ref="PSR2">
+    <!-- Allow closing braces for functions and classes to be on the same line -->
+    <exclude name="Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine"/>
+    <exclude name="PSR2.Classes.ClassDeclaration.OpenBraceNewLine"/>
+    <exclude name="Squiz.WhiteSpace.ScopeClosingBrace"/>
```
Comment (montchr) [inline ruleset.xml]: These exclusions affect opening braces too. Should comment be changed?

Comment (retlehs) [inline ruleset.xml]: @montchr good catch, mind submitting a PR?

Comment (montchr) [inline ruleset.xml]: @retlehs Sure. I can do that today.

**Pull request merged**